vector/src/main/java/im/vector/app/features/spaces/NewSpaceSummaryItem.kt

```diff
@@ -34,16 +35,25 @@ import org.matrix.android.sdk.api.util.MatrixItem
 abstract class NewSpaceSummaryItem : VectorEpoxyModel<NewSpaceSummaryItem.Holder>(R.layout.item_new_space) {
 
     @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) lateinit var avatarRenderer: AvatarRenderer
+    @EpoxyAttribute var countState: UnreadCounterBadgeView.State = UnreadCounterBadgeView.State(0, false)
+    @EpoxyAttribute var expanded: Boolean = false
+    @EpoxyAttribute var hasChildren: Boolean = false
     @EpoxyAttribute lateinit var matrixItem: MatrixItem
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var onLongClickListener: ClickListener? = null
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var onSpaceSelectedListener: ClickListener? = null
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var onToggleExpandListener: ClickListener? = null
     @EpoxyAttribute var selected: Boolean = false
-    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var listener: ClickListener? = null
-    @EpoxyAttribute var countState: UnreadCounterBadgeView.State = UnreadCounterBadgeView.State(0, false)
 
     override fun bind(holder: Holder) {
         super.bind(holder)
-        holder.rootView.onClick(listener)
+        holder.root.onClick(onSpaceSelectedListener)
+        holder.root.setOnLongClickListener { onLongClickListener?.invoke(holder.root).let { true } }
```

Comment:
`.let { true }` is maybe not useful. `it` is not used.
Having `return true` is probably better.